services/meta/service_test.go

```diff
@@ -205,3 +477,75 @@ func mustParseStatement(s string) influxql.Statement {
 	}
 	return stmt
 }
+
+// Test long poll of snapshot.
```

Comment:
ah...yeah, I had removed it originally.  That just didn't survive the rebase.  :)  I'll remove it.